app/models/project.rb

```diff
@@ -166,6 +166,67 @@ def to_salesforce_json
                 json.set!('username', self.user.email)
             end
             json.application do
+                json.set!('projectName', self.project_title)
+                json.projectDateRange do
+                    json.startDate self.start_date
+                    json.endDate self.end_date
+                end
+                json.projectAddress do
+                    json.line1 self.line1
+                    json.line2 self.line2
+                    json.line3 self.line3
+                    json.county self.county
+                    json.townCity self.townCity
+                    json.projectPostcode self.postcode
+                end
+                json.set!('yourIdeaProject', self.description)
+                json.set!('projectDifference', self.difference)
+                json.set!('projectCommunity', self.matter)
+                json.set!('projectOrgBestPlace', self.best_placed_description)
+                json.set!('projectAvailable', self.heritage_description)
+                json.set!('projectOutcome1', 'Dummy data')
```

Comment:
```suggestion
                json.set!('projectOutcome1', self.involvement_description)
```
This is the projectOuctcome1 question

Reply:
Can we update the test to include this field?